code/src/ui/interaction/tree/info/time-series-panel.tsx

```diff
@@ -32,8 +32,8 @@ export default function TimeSeriesPanel(props: Readonly<TimeSeriesPanelProps>) {
   return (
     <div className={styles["time-series-panel"]}>
       <DropdownField options={parseTimeSeriesIntoOptions(tsData)} selectedIndex={selectedTimeSeriesOption} setSelectedIndex={setSelectedTimeSeriesOption} />
-      <Chart data={tsData} selectedIndex={selectedTimeSeriesOption} />
-      <Table group={tsData} selectedIndex={selectedTimeSeriesOption} />
+      <Chart data={tsData[selectedTimeSeriesOption]} />
+      <Table data={tsData[selectedTimeSeriesOption]} />
```

Comment:
```suggestion
      <Table data={tsData[selectedTimeSeriesOption]} />
```
Let's remove this line and not display the raw data

Reply:
Please check commit https://github.com/TheWorldAvatar/viz/pull/100/commits/81cd34f4d5fd62cdbfef8b98c4ebecfb7d211516